config/car.csv

```diff
@@ -0,0 +1,5 @@
+zero123_weight, radius, theta, phi, image
```

Comment:
personally I prefer `polar` and `azimuth` instead of `theta` and `phi`. @ashawkey I'm not saying we should rename them everywhere, but since this csv file will be human-authored, we should stick with names that are easily understood and remembered...

Reply:
I agree, `polar/azimuth` may be better.

Reply:
Replaced everywhere :)